pages/_app.js

```diff
@@ -1,10 +1,23 @@
 import GlobalStyle from "../styles";
+import useLocalStorageState from "use-local-storage-state";
+import data from "@/db/data";
 
 export default function App({ Component, pageProps }) {
+  const [recipes, setRecipes] = useLocalStorageState("recipes", {
+    defaultValue: [...data],
```

Comment:
```suggestion
    defaultValue: data
```

Should do the same, shouldnt it?